pam/deploy_template.yaml

```diff
@@ -0,0 +1,255 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: approval-pam
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: approval-pam
+    labels:
+      app: approval
+    annotations:
+      description: All the KIE server web server's ports.
+  spec:
+    ports:
+    - name: approval-pam
+      port: 8080
+      targetPort: 8080
+    selector:
+      name: approval-pam
+- kind: Service
+  apiVersion: v1
+  spec:
+    clusterIP: "None"
```

Comment:
I don't understand this. Why would we create a headless service for a ping endpoint?

Reply:
I don't understand either. It seems the PAM inside is expecting this service. I saw the ENV
```
          - name: OPENSHIFT_DNS_PING_SERVICE_NAME
            value: approval-pam-ping
          - name: OPENSHIFT_DNS_PING_SERVICE_PORT
            value: "8888"
```